File: php/backups.php
<?php

require_once 'entity.php';

class Backup extends Entity
{
    public static $resource = '/backups/';
    public static $json_name = 'backup';

    public function __construct() {
        parent::__construct();
        $this->json_property('id', 'integer');
        $this->json_property('file', 'File', '',
                             false, $recursive = True);
    }
}

class Backups extends Resource
{
    public function __construct() {
        $this->entity_class = 'Backup';
        $this->json_name = 'backups';
    }
}

$backups = new Backups();

File: python/backups.py
import sdk.python.entities
from sdk.python.files import File
from sdk.python.entities import Property


class Backup(sdk.python.entities.Entity):

    resource = '/backups/'
    json_name = 'backup'

    id = Property(int)
    file = Property(File)


class Backups(sdk.python.entities.Resource):

    entity_class = Backup
    json_name = 'backups'


backups = Backups()
